modules/core/test/com/haulmont/cuba/core/MessagesTest.java

```diff
@@ -111,6 +112,26 @@ public void testMultiInclude() {
         assertEquals(msg3, "overridden");
     }
 
+    @Test
+    public void testIncludeDefaultLocale() {
+        Messages messages = AppBeans.get(Messages.class);
+        UserSessionSource userSessionSource = AppBeans.get(UserSessionSource.NAME);
+        UserSession userSession = userSessionSource.getUserSession();
+        Locale defaultLocale = userSession.getLocale();
+        try {
+            String msg = messages.getMessage("com.haulmont.cuba.core.mp_test", "includedMsg");
+            assertEquals("Included Message", msg);
+
+            userSession.setLocale(Locale.forLanguageTag("ru"));
+            messages.clearCache();
+            msg = messages.getMessage("com.haulmont.cuba.core.mp_test", "includedMsg");
```

Comment:
If you change the package path to the correct one (**com.haulmont.cuba.core.mp_test.inc**), this test will also work without your fix

Reply:
Yes, but in this case include key `@include=com.haulmont.cuba.core.mp_test.inc` is not used. The issue and the fix are about include keys.